libs/ui/src/containers/screens/contact-update-screen/ContactUpdateScreen.fetcher.tsx

```diff
@@ -0,0 +1,11 @@
+import {
+  GetContactForm,
+  getContactFormWidgetInitialData,
+} from '../../widgets/contact-form-widget/ContactFormWidget.fetcher';
+
+export async function getContactUpdateScreenProps(
+  id: number
+): Promise<GetContactForm['initialData']> {
+  const contact = await getContactFormWidgetInitialData(id);
+  return contact;
+}
```

Comment:
ini yang direturn langsung semua props nya aja ya drik : 

```suggestion
export async function getContactUpdateScreenProps(
  id: number
): Promise<ContactUpdateScreenProps> {
  const contact = await getContactFormWidgetInitialData(id);
  return { contact, id };
}
```

Reply:
Disitu yang direturn langsung type `ContactUpdateScreenProps`